cmd/traefik/traefik.go

```diff
@@ -75,7 +75,7 @@ Complete documentation is available at https://traefik.io`,
 		os.Exit(-1)
 	}
 
-	if _, err := f.Parse(usedCmd); err != nil {
+	if _, err = f.Parse(usedCmd); err != nil {
 		if err == pflag.ErrHelp {
 			os.Exit(0)
 		}
```

Comment:
Unrelated to this PR: I think line 83 should return an exit code of `1`; negative exit codes are at least very uncommon, if not possibly also troublesome.